src/main/java/com/example/training/config/ModelMapperConfig.java

```diff
@@ -0,0 +1,18 @@
+package com.example.training.config;
+
+
+import org.modelmapper.ModelMapper;
+import org.modelmapper.convention.MatchingStrategies;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ModelMapperConfig {
+    @Bean
+    public ModelMapper modelMapper() {
+        ModelMapper modelMapper = new ModelMapper();
+        modelMapper.getConfiguration()
+                .setMatchingStrategy(MatchingStrategies.STRICT);
+        return modelMapper;
+    }
+}
```

Comment:
End of file. Để 1 dòng trống cuối file
```suggestion
}

```